content/docs/Concepts/versioning.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Versioning"
+linkTitle: "Versioning"
+weight: 3
+date: 2020-04-10
+description: >
+  OpenCue policy on version number changes and compatibility between versions.
+---
+
+This page defines OpenCue policy on version number changes and compatibility between versions.
+
+OpenCue version numbers contain three components: `<major>.<minor>.<patch>`.
+
+- Major version will be incremented only be agreement of TSC. Major versions will be rare and
```

Comment:
Major version will be incremented only **by** agreement of the technical steering committee (TSC).

(Fix a typo and expand acronym on first use)

Reply:
Done.